utils/data.py

```diff
@@ -522,30 +465,7 @@ def to_cpu(motion_data):
     return motion_data
 
 
-def un_normalize(data, mean, std):
-    if isinstance(data, list):
-        for i in range(len(data)):
-            data[i] = un_normalize(data[i], mean, std)
-    else:
-        if data.shape[1] == std.shape[1]:
-            data = data * std + mean
-    return data
-
-
-def is_list_4D(data):
-    return isinstance(data, list) and all([d.ndim==4 and d.shape[0] == 1 for d in data])
-
-
-def anim_from_edge_motion_data(edge_motion_data, edge_rot_dict_general):
-    edge_rots_dict, _, _ = edge_rot_dict_from_edge_motion_data(edge_motion_data, edge_rot_dict_general=edge_rot_dict_general)
-    n_motions = edge_motion_data.shape[0]
-    anims = [None]*n_motions
-    for idx, one_edge_rot_dict in enumerate(edge_rots_dict):
-        anims[idx], names = anim_from_edge_rot_dict(one_edge_rot_dict)
-    return anims, names
-
-
-def anim_from_edge_rot_dict(edge_rot_dict, root_name='Hips'):
+def basic_anim_from_rot(edge_rot_dict, root_name='Hips'):
     assert root_name in edge_rot_dict['names_with_root']
     root_idx = np.where(edge_rot_dict['names_with_root'] == root_name)[0][0]
```

Comment:
when is this code used?

Reply:
during sanity check in pre preocess, I will look it up trying to change it to use staticData also.